Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -21,18 +21,76 @@ almost always find the directory structure as outlined below.
 Also see :ref:`Environment` for further information, especially how to retrieve
 the paths within PHP code.
 
+Files on project level
+======================
+
+On the top most level, the project level, you can find the files
+:file:`composer.json` which contains requirements for the TYPO3 installation
+and the :file:`composer.lock` which contains information about the concretely
+installed versions of each package.
+
 Directories in a typical project
 ================================
 
 .. contents::
    :local:
 
+.. _directory-config:
+
+:file:`config/`
+---------------
+
+Composer installations only. TYPO3 configuration directory. This directory
+contains installation-wide configuration.
+
+.. _directory-config-sites:
+
+:file:`config/sites/`
+~~~~~~~~~~~~~~~~~~~~~
+
+Composer installations only.  The folder :file:`config/sites` contains
+subfolders for each :ref:`site configuration <sitehandling>`.
+
+
+.. _directory-local_packages:
+
+:file:`local_packages/`
+-----------------------
+
+Composer installations only.  Each web site which is run on TYPO3 **should**
+have a sitepackage, anextension with a special purpose containing all
+templates, styles, images, etc. needed for the theme.
+
+It is usually stored locally and then symlinked into the :ref:`directory-vendor`
+folder. Many projects also need custom extensions that can be stored here.
+
+The folder for local packages has to be defined in the project's :file:`composer.json`
+to be used:
+
+..  code-block:: json
+    :caption: composer.json
+    {
+        "name": "myvendor/my-project",
+        "repositories": {
+            "my_local_packages": {
+                "type": "path",
+                "url": "local_packages/*"
+            }
+        },
+        "...": "..."
+    }
+
+.. _directory-public:
 
 :file:`public/`
 ---------------
 
-We assume here that your web root points to a folder called :file:`public` in a Composer-based installation or a legacy 
-installation as is commonly done. Otherwise, replace  :file:`public` with the path to your web root.
+We assume here that your web root points to a folder called :file:`public` in
+a Composer-based installation or a legacy installation as is commonly done.
+Otherwise, replace  :file:`public` with the path to your web root. In legacy
+installations by default all files are located in this folder.
```

Comment:
"In legacy installations, all files are located in this folder by default."

Reply:
will be solved in a follow up